lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -374,3 +369,17 @@ def _detach_deleted_subscribers_done(self, delete_future):
                 "Detach Deleted Subscribers Error! [%s] %s",
                 err.code(), err.details(),
             )
+
+
+def _log_grpc_error(err: grpc.RpcError) -> None:
+    if err.code() == grpc.StatusCode.INTERNAL:
+        logging.error(
+            'Internal Error in subscriberdb-Orc8r communication. '
+            'If AGW and Orc8r are deployed at a commit hash, '
+            'upgrade AGW and Orc8r to latest commit hash.',
+        )
+    else:
+        logging.error(
+            "%s request error! [%s] %s", inspect.stack(0)[1][3], err.code(),
```

Comment:
inspect.stack(0)[1][3] can we annotate what the indecies mean here? (I'm guessing it get's the name out?)

Reply:
Yes, it is. I added added a comment to explain.